pandas/core/series.py

```diff
@@ -2227,20 +2229,31 @@ def combine(self, other, func, fill_value=np.nan):
         Series.combine_first : Combine Series values, choosing the calling
             Series's values first
         """
+        self_is_ext = is_extension_array_dtype(self.values)
+        if fill_value is None:
+            fill_value = na_value_for_dtype(self.dtype, False)
```

Comment:
Use the keyword for False so it’s clearer what’s being specified.

Reply:
OK, that is done.